CorsixTH/Lua/epidemic.lua

```diff
@@ -261,16 +262,22 @@ function Epidemic:checkNoInfectedPlayerHasLeft()
     if infected_patient.going_home and not infected_patient.cured and
         px and py and not self.hospital:isInHospital(px,py) then
       -- Patient escaped from the hospital, discovery is inevitable.
-      self.result_determined = true
-      if not self.inspector then
-        self:spawnInspector()
-      end
       self:finishCoverUp()
       return
     end
   end
 end
 
+function Epidemic:checkThereAreNoAnyInfectedPatientsLeft()
```

Comment:
```suggestion
function Epidemic:checkNoInfectedPatients()
```

Function needs some Lua documentation.

Reply:
> Function needs some Lua documentation.

Added as for other functions in this file